hack/binaries

```diff
@@ -18,15 +18,15 @@ binariesLegacy() {
 
   case "$(echo "$TARGETPLATFORM" | cut -d"/" -f1)" in
   "darwin")
-    docker build --iidfile $iidfile --target buildctl-darwin -f ./hack/dockerfiles/test.Dockerfile --force-rm .
+    docker build --iidfile $iidfile --build-arg BUILDKITD_TAGS="${BUILDKITD_TAGS}" --target buildctl-darwin -f ./hack/dockerfiles/test.Dockerfile --force-rm .
```

Comment:
nit: Don't really need it in darwin as buildkitd is not built there

Reply:
ah! my mistake; let me remove (I'll rebase the other PR)

Reply:
fixed